user-management.yaml

```diff
@@ -388,6 +394,66 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/GenericError'
+         
+  /users/{username}/force:            
```

Comment:
We should maybe refactor this in the future, a DELETE /users/{username}?force=true, i.e. query parameter would be cleaner. In REST convention, we would expect to delete the force of a user here

Reply:
Issue is created
https://github.com/upb-uc4/lagom-core/issues/393